django_app/settings.py

```diff
@@ -75,6 +83,31 @@
 
 WSGI_APPLICATION = "django_app.wsgi.application"
 
+AUTHENTICATION_BACKENDS = [
+    # Needed to login by username in Django admin, regardless of `allauth`
+    'django.contrib.auth.backends.ModelBackend',
+
+    # `allauth` specific authentication methods, such as login by e-mail
+    'allauth.account.auth_backends.AuthenticationBackend',
+]
+
+# db id of the entry in 'Sites' table
+SITE_ID = 2
+
+LOGIN_REDIRECT_URL = '/polls/'
+LOGOUT_REDIRECT_URL = '/polls/'
+
+# Provider specific settings allauth
+SOCIALACCOUNT_PROVIDERS = {
+    'google': {
+        'APP': {
+            'client_id': config('GOOGLE_CLIENT_ID'),
```

Comment:
Why don't we follow the convention in the rest of the file for selecting env variables?
This might break digital ocean. Follow existing conventions.

Reply:
I mean this is usually recommended to use .env file to manage env vars but I will follow existing convention